jenkins/ci.suse.de/cloud-ardana8.yaml

```diff
@@ -9,18 +9,6 @@
     jobs:
         - '{cloud_url_trigger_job}'
 
-- project:
-    name: cloud-ardana8-updates-test-trigger
-    cloud_url_trigger_job: '{name}'
-    version: '8'
-    url: 'http://provo-clouddata.cloud.suse.de/repos/x86_64/SUSE-OpenStack-Cloud-{version}-Updates-test/SUSE:Maintenance:Test:OpenStack-Cloud:{version}:x86_64.repo'
-    check_date: true
-    projects:
-      - project: 'cloud-ardana{version}-job-entry-scale-kvm-pcloud001-x86_64'
```

Comment:
This job is still defined below, did you also want to remove that?

Reply:
I missed it, thank you!